velox/expression/SwitchExpr.cpp

```diff
@@ -86,17 +83,23 @@ void SwitchExpr::evalSpecialForm(
       }
     }
   }
+
+  VectorPtr condition;
+  const uint64_t* values;
+
   for (auto i = 0; i < numCases_; i++) {
+    context.releaseVector(condition);
```

Comment:
nit: can you update the description as the old one no longer applies.

Reply:
you mean the PR summary? updated in phab already